<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkeletonMergingWidget</class>
 <widget class="QDialog" name="SkeletonMergingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Attempt to merge:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sourceSkeletonComboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>into</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="targetSkeletonName">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Common bones:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="commonBonesList">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="additiveMergeButton">
       <property name="text">
        <string>Additive Merge</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ogreMergeButton">
       <property name="text">
        <string>Ogre merge</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SkeletonMergingWidget</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
